grammars.yml

```diff
@@ -616,6 +616,9 @@ vendor/grammars/sourcepawn:
 - source.sp
 vendor/grammars/sql.tmbundle:
 - source.sql
+vendor/grammars/squirrel-language:
+- source.nut
+- source.nut
```

Comment:
This last line needs to be removed. It's [causing](https://travis-ci.org/github/linguist/jobs/267676997#L1262) the build to fail, due to the duplicated scope.